src/components/header.js

```diff
@@ -5,7 +5,11 @@ import "../styles/header.css"
 function Header() {
     return (
         <div className="header">
-            Header
+            <ul className="header-ul">
```

Comment:
You already know that this is `ul` from the tag name, no need to specify it in the class name.